communitiesuk/funding-service-design-assessment
# FS-3873: Cookies page and banner on assessment

https://dluhcdigital.atlassian.net/browse/FS-3873
https://dluhcdigital.atlassian.net/browse/FS-3874

### Change description
- Cookie Policy page added
- Route to cookie policy page added
- Cookie banner added with accept and reject options
- Unused JS functions removed
- Show all sections and show are being rendered by the govuk frontend in the version being used



### How to test
- Go to the assessment dashboard, you should be prompted to accept or reject the cookies in the form of a banner.
- It should retain info what choice you made for the duration of your session
- Observe the details in the dev tools to see cookies being created
- Verify you are able to visit the cookies page, read and make a choice from there as well


### Screenshots of UI changes (if applicable)

<img width="1358" alt="Screenshot 2024-06-24 at 06 05 32" src="https://github.com/communitiesuk/funding-service-design-assessment/assets/66220499/0868561d-6464-4faa-9859-fa8bd2a80764">

<img width="1156" alt="Screenshot 2024-06-24 at 06 05 58" src="https://github.com/communitiesuk/funding-service-design-assessment/assets/66220499/392c750d-0c64-45a9-b45d-822a77a3e2f8">

<img width="1232" alt="Screenshot 2024-06-24 at 06 06 18" src="https://github.com/communitiesuk/funding-service-design-assessment/assets/66220499/be4b738a-78bc-4dd2-94a8-40a6124d5aa9">
